README.md

```diff
@@ -334,8 +334,8 @@ while !search_queue.empty?
     next
   end
 
-  # add back the search
-  search_queue.push(search)
+  # add result to the search queue
+  search_queue.push(result)
```

Comment:
Good job find this @waynehoover and thanks for the fix! 👍

We can merge this directly as the failing tests are probably unrelated.